new-games-to-db/main.py

```
@@ -98,7 +97,7 @@ def new_games_to_db(request):
     help='Overwrite mismatched values if the only difference is the year')
   p.add_argument('-n', '--dry_run', default=False, action='store_true')
   p.add_argument('--nodry_run', dest='dry_run', action='store_false')
-  r = p.parse_args()
+  r = p.parse_args(args=args)
```

Pylint (3.3.7)

[warning] Bad indentation. Found 2 spaces, expected 4

(W0311)